### DISCUSSION TITLE
Birdseye in 0.14

### DISCUSSION THREAD

**Discussion by mikje** [blakeblackshear/frigate#13177] [General]
In frigate 13 I see live streams from my cams under birthview. It seems that this is removed in version 14.
I read that in 14 live streams are automatically started when movement is detected.
I want it always a live stream. Is this still possible in v 14 and how?

**Comment by hawkeye217** [comment]
Birdseye is still available in 0.14.

See the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 3.

**Reply by mikje** [reply]
yes but it is not streaming constantly , my camera times in the views are not counting, only when events are triggering the cameras. In vers 0.13 it was always streaming. And i want that same situation.

**Reply by hawkeye217** [reply]
See the FAQ, Usage question 5.

If you want a continuous stream, use Birdseye on `continuous` mode and add Birdseye to a camera group.

**Reply by NickM-27** [reply]
> In vers 0.13 it was always streaming

that is not correct, in 0.13 the dashboard did not support streaming at all. You can add birdseye as a custom camera group if you want to. 

Continuous streaming of all cameras that are not detecting any motion is not supported, because it would look exactly the same as it does currently except using more server resources, network bandwidth, etc.

there are a lot of optimizations that ensure the camera updates very quickly so there is no latency

**Reply by mikje** [reply]
my cameras show the time in the picture. And the time was ticking in seconds in bv in version 13. In version 14 this is not the case. How come?

**Reply by NickM-27** [reply]
if you were on the main dashboard in 0.13 frigate UI that showed multiple cameras, it is impossible that the camera images were updating because that was not implemented

**Reply by mikje** [reply]
the main dashboard and then i choose birdseye  then i see all my cams in mini format and all are updating every second the time

**Reply by NickM-27** [reply]
yes, and you can still do that in 0.14

**Reply by mikje** [reply]
But in 14 my camera's are not streaming continuously in bv because I see that at the seconds that are standing still

**Reply by NickM-27** [reply]
the behavior of birdseye has not changed between versions

**Reply by hawkeye217** [reply]
> But in 14 my camera's are not streaming continuously in bv because I see that at the seconds that are standing still

If it's not already there put this in your frigate config file:
```
birdseye:
  mode: continuous
```

Then navigate to `http://<frigate_ip>:5000/#birdseye`.

As @NickM-27 said, this behavior has not changed in 0.14. If you are not seeing birdseye working correctly, then something else is wrong with your installation.

**Reply by mikje** [reply]
Strange that it seems to me there is a difference

**Reply by hawkeye217** [reply]
Can you show a screen recording of what you are seeing? Nobody else has reported a problem with Birdseye in 0.14.

**Comment by mikje** [comment]
well now it seems to work. If I have the issue again I will add the recordings

**Comment by mikje** [comment]
Here you see that version 13 is streaming continuously and 14 is not

https://github.com/user-attachments/assets/a27089b6-7cb1-4f5b-9717-cd3006ffa4d7


https://github.com/user-attachments/assets/20934cd6-97b2-4ed4-8b9e-4778abd471f6

**Reply by hawkeye217** [reply]
We’ll need to see your frigate config file and logs.

**Comment by mikje** [comment]
[_frigate14_logs (1).txt](https://github.com/user-attachments/files/16710304/_frigate14_logs.1.txt)
[fg_conf.txt](https://github.com/user-attachments/files/16710399/fg_conf.txt)

**Reply by hawkeye217** [reply]
Are you running via docker? If so, what is your docker compose?

**Reply by mikje** [reply]
# version: '3.9'

services:

  frigate:
    container_name: frigate14
    privileged: true
    restart: unless-stopped
    image: ghcr.io/blakeblackshear/frigate:stable
    shm_size: "128mb" # update for your cameras based on calculation above
    devices:
      - /dev/bus/usb:/dev/bus/usb
      - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your ha
rdware
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /opt/frigate14/config:/config
      - /cctv:/media/frigate
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5000:5000"
      - "8971:8971"
      - "8554:8554"
      - "8555:8555"
    environment:
      FRIGATE_RTSP_PASSWORD: "pwd"

**Comment by mikje** [comment]
[_frigate14_logs.1.txt](https://github.com/user-attachments/files/16715797/_frigate14_logs.1.txt)

**Comment by mikje** [comment]
i added new logs after restarting. And still not continuous play in birdeye, while this was in version 13 playing continuously

**Comment by mikje** [comment]
I run frigate in a docker and i have both version 13 and 14 in seperate containers. The errors in the first logfile where due to not restarting the proxmox docker vm.

**Comment by hawkeye217** [comment]
Can you post a screenshot of your system metrics general and camera pages from the UI?

**Comment by mikje** [comment]
![Screenshot 2024-08-22 201636](https://github.com/user-attachments/assets/af2d1ba7-027a-4f33-b113-51332581edc4)
![Screenshot 2024-08-22 201658](https://github.com/user-attachments/assets/5e28993b-267d-4857-a21c-a4503e1aef3b)
![Screenshot 2024-08-22 201717](https://github.com/user-attachments/assets/7090f4a2-e9aa-488a-abab-1ac40e866f09)
![Screenshot 2024-08-22 201739](https://github.com/user-attachments/assets/144ea7e4-498f-486d-b0cb-c8279c1ca6f1)
![Screenshot 2024-08-22 201523](https://github.com/user-attachments/assets/83a48c93-e434-41ab-994b-18676873b33e)
![Screenshot 2024-08-22 201616](https://github.com/user-attachments/assets/a9404dd7-5226-48db-80a1-b154ec09135a)

**Comment by mikje** [comment]
and 2 more for storage 

![Screenshot 2024-08-22 201942](https://github.com/user-attachments/assets/cf834246-fd69-420a-9ab3-7aaa51fe5fd9)
![Screenshot 2024-08-22 201956](https://github.com/user-attachments/assets/b6b4dd76-1c17-42e3-a5d5-3b418f101bf9)

**Comment by hawkeye217** [comment]
You have many skipped detections on your cameras. This indicates that your detection pipeline is being overloaded, and this is why birdseye is failing at times. You should use the motion tuner in the UI for your cameras to tune your motion settings, and you should reduce your detect fps to 5. Degraded performance is typical with values above that for many cameras.

**Comment by mikje** [comment]
"You should use the motion tuner in the UI for your cameras to tune your motion settings"  where can i find this or do you mean in the camera setup outside of frigate?

**Reply by hawkeye217** [reply]
Settings --> Motion Tuner tab

**Comment by mikje** [comment]
thx found it, but all of my cameras show:   
![image](https://github.com/user-attachments/assets/7b083271-10cf-4882-88b2-ac78be5fd8cf)

**Reply by hawkeye217** [reply]
Your system can't process frames because of all of the motion or activity on your cameras at the moment, so I would recommend disabling some of your cameras temporarily and tuning the motion separately. While you're in your config file, I'd recommend lowering the `detect` `fps` values to 5.

**Reply by mikje** [reply]
Thx

**Comment by mikje** [comment]
Is adding more cpu and ram to the docker also an idea?

**Reply by hawkeye217** [reply]
That could potentially help, but lowering your `fps` would be best.

**Comment by mikje** [comment]
fps 5 makes the be working for me and streaming constantly. Thx so far for all helpfull hints. Will still have a look at the motion det. tuner  to optimize

**Reply by hawkeye217** [reply]
Great. Happy to help.

**Comment by mikje** [comment]
the motion trigger that it looks like streaming is that the motion is triggered by the seconds from the clock of the camera stream :-)

**Reply by NickM-27** [reply]
a motion mask should be added to cover the timestamp
